src/tfrobot/tfrobot_config.md

```diff
@@ -0,0 +1,115 @@
+<h1> Configuration File</h1>
+
+<h2>Table of Contents</h2>
+
+- [Introduction](#introduction)
+- [Examples](#examples)
+  - [YAML Example](#yaml-example)
+  - [JSON Example](#json-example)
+- [Create a Configuration File](#create-a-configuration-file)
+
+***
+
+## Introduction
+
+To use TFROBOT, the user needs to create a YAML or a JSON configuration file that will contain the mass deployment information, such as the groups information, number of VMs to deploy how, the compute, storage and network resources needed, as well as the user's credentials, such as the SSH public key, the network (main, test, dev, qa) and the TFChain mnemonics.
+
+## Examples
+
+We present here a configuration file example that deploys a 3 nodes, with 2 vcores, 16GB of RAM, 100GB of SSD and 50GB of HDD. The same deployment is shown with a YAML file and with a JSON file. Parsing is based on file extension, TFROBOT will use JSON format if the file has a JSON extension and YAML format otherwise.
+
+You can use this example for guidance, and make sure to replace placeholders and adapt the groups based on your actual project details. To the minimum, `ssh_key1` should be replaced by the user SSH public key and `example-mnemonic` should be replaced by the user mnemonics.
+
```

Comment:
Can you also add any kind of IPs in the example (ygg, ipv4, or ipv6)?

Reply:
Done. Added ipv4 true, ipv6 false.